compiler/vmops.nim

```diff
@@ -159,13 +160,17 @@ proc registerAdditionalOps*(c: PCtx) =
     setResult a, c.config.projectPath.string
 
   wrap1f_math(sqrt)
+  wrap1f_math(cbrt)
   wrap1f_math(ln)
   wrap1f_math(log10)
   wrap1f_math(log2)
   wrap1f_math(exp)
   wrap1f_math(arccos)
   wrap1f_math(arcsin)
   wrap1f_math(arctan)
+  wrap1f_math(arcsinh)
```

Comment:
in future work, we should be able to write:
`wrap(arcsinh)` and it'd automatically do the conversion based on proc declaration